tests/vm/tcompiletimetable.nim

```diff
@@ -1,12 +1,16 @@
 discard """
-  nimout: '''2
+  nimout: '''
+2
 3
 4:2
 Got Hi
 Got Hey
+'''
+  output:'''
```

Comment:
original tests are wrong. `nimout` only compares compile messages.

Reply:
yikes, is that yet another bug than https://github.com/nim-lang/Nim/issues/16701 ?